component.mk

```diff
@@ -19,8 +19,10 @@ HOMEKIT_MAX_CLIENTS ?= 16
 # Set to 1 to enable WolfSSL low resources, saving about 70KB in firmware size,
 # but increasing pair verify time from 0.5 to 3.6 secs.
 HOMEKIT_SMALL ?= 0
-# Set to 1 to disable overclock on pair-verify function (Will increase pair verify time to double).
-HOMEKIT_DISABLE_OVERCLOCK ?= 0
+# Set to 1 to enable overclock on initial pair-setup function (It will reduce pair setup time by half).
```

Comment:
Let's make a top-level switch - HOMEKIT_OVERCLOCK - that will enable/disable all overclocking options at once and then have per-stage options.. Also, please rename HOMEKIT_OVERCLOCK_SETUP -> HOMEKIT_OVERCLOCK_PAIR_SETUP and same for verify. HOMEKIT_OVERCLOCK might just be a Makefile variable, no need to have it in *.c files.

Reply:
Changes done.